src/overlays/actors/ovl_En_Peehat/z_en_peehat.h

```diff
@@ -4,11 +4,48 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define GROUND_HOVER_HEIGHT 75.0f
+
+#define MAX_LARVA 3
+#define TYPE_GROUNDED -1
+#define TYPE_FLYING 0
+#define TYPE_LARVA 1
+
 struct EnPeehat;
 
+//params -1 = Ground Peahat
+//params 0 = Spawner Peahat
+//params 1 = Peahat larva
```

Comment:
I would make this a type enum, remove the defines above, and remove the comments

Reply:
and id prefix them with PEAHAT_TYPE_